frontend/app/ui/subscriptions/detail/controller.js

```diff
@@ -25,19 +25,35 @@ export default class SubscriptionsDetailController extends Controller {
       const reports = yield this.timed.getProjectReports(this.reportsPage++);
       this.reports.pushObjects(reports.toArray());
 
-      this.reportsNext = get(reports, "links.next");
+      this.reportsNext = Boolean(get(reports, "links.next"));
     } catch (error) {
       console.error(error);
       this.reportsNext = false;
     }
   }
 
+  @task *fetchNextOrders() {
+    try {
+      const orders = yield this.timed.getProjectOrders(this.ordersPage++);
```

Comment:
`a++` returns 1 if a === 1. `++a` returns 2 if a === 1
```suggestion
      const orders = yield this.timed.getProjectOrders(++this.ordersPage);
```

Reply:
I would also prefer this to be on a line on its own since `method(++i)` is a bit confusing to read. Or `this.ordersPage += 1;`.
But this is a minor nitpick.

Link to a discussion about this: https://stackoverflow.com/questions/971312/why-avoid-increment-and-decrement-operators-in-javascript

Reply:
Good catch! :sunglasses: 

I'd rather just correct the mistake but after just having made the error I can't really argue :smile: 

Changed it.